tools/functional-tester/etcd-tester/main.go

```diff
@@ -49,6 +51,8 @@ func main() {
 	consistencyCheck := flag.Bool("consistency-check", true, "true to check consistency (revision, hash)")
 	isV2Only := flag.Bool("v2-only", false, "'true' to run V2 only tester.")
 	stresserType := flag.String("stresser", "default", "specify stresser (\"default\" or \"nop\").")
+	failureTypes := flag.String("failures", "default,failpoints", "specify failures (concat of \"default\" and \"failpoints\").")
+	terminate := flag.Bool("terminate", true, "'true' to terminate an etcd cluster after testing.")
```

Comment:
`'true' to terminate the test cluster on exit.`

Is it necessary to have this flag / is there another way to get this functionality? Having a flag that amounts to leaking server resources seems bad because the cleanup will have to be duplicated elsewhere. What's the use case / workflow?